CMakeLists.txt

```diff
@@ -2828,6 +2817,64 @@ elseif(WIN32)
   endif()
 endif()
 
+# Mixxx translation files
+file(GLOB MIXXX_TS_FILES "${CMAKE_CURRENT_SOURCE_DIR}/res/translations/*.ts")
+message(VERBOSE "Mixxx translation files: ${MIXXX_TS_FILES}")
+
+if(QT6)
+  find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS LinguistTools)
+  set_source_files_properties(
+    ${MIXXX_TS_FILES}
+    PROPERTIES OUTPUT_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/res/translations"
```

Comment:
Normally, if we compile a source file, we store the output in `${CMAKE_CURRENT_BINARY_DIR}`, Why storing the .qm files in the source directory instead?

Reply:
> Why storing the .qm files in the source directory instead?

Because otherwise I guess it won't be picked up when compiling Mixxx locally and then running it directly from the build dir. Personally I don't really care (using `LANG=en_US.UTF8` anyway), but I think makes sense to be able to test localization without installing Mixxx.

Reply:
We have already a hack to find the resource files from a Mixxx source tree.

https://github.com/mixxxdj/mixxx/blob/a716202b0b5bd6923d4d4438d62dba5060a19791/src/preferences/configobject.cpp#L46

We may handele the translations separate to load them from the build tree.

Reply:
Sure, sounds like a good follow up to this PR \*wink-wink\*.